.github/workflows/scripts/sensitive_file_check.py

```
@@ -0,0 +1,123 @@
+
+    Args:
+        config_file: Path to the configuration file containing regex patterns.
+
+    Returns:
+        List[str]: A list of regex patterns.
+    """
+    if not os.path.exists(config_file):
+        print(f"Error: Configuration file '{config_file}' not found.")
+        sys.exit(1)
+
+    with open(config_file, "r") as f:
```

Ruff (0.14.11)

Unnecessary mode argument

Remove mode argument

(UP015)